README.md

```diff
@@ -35,4 +35,17 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like developement mode (which also enables features like hot reloading when you make a file change).
 * There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
-When running `setup.sh`, the `.env.template` file will be copied to `.env` if the latter does not exist.
\ No newline at end of file
+When running `setup.sh`, the `.env.template` file will be copied to `.env` if the latter does not exist.
+
+Following is the link of Trello account 
+
+https://trello.com/b/smcVvamd/book-lib 
+
+Details below for the trello boards details:
+
+BOARD_ID=5f58a4a02301b638351b3bce
+KEY=f16aee2c005a4d41c2e7252416e20a9a
+TOKEN=0b8ca8a685ea8c7ef812f92680257233f9eabe98da53afa83762db4e86a41666
+LISTDO=5f58a4a10ca5a331359ddba3
+LISTDONE=5f58a4a10b7b403852fbeace
```

Comment:
You should never include credentials in the codebase! And there's no need to include the board details because e.g. developers could each have their own board to test against without interfering with each other.